data/example-data-oracle-book/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
copy right should be at the very top

Reply:
Not able to put copyright above xml declaration.